_posts/2020-10-19-intro-to-hive-connector.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "A gentle introduction to the Hive connector"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+One of the most confusing aspects when starting Presto is the Hive connector.
+
+TL;DR: The Hive connector is what you use in Presto for querying object storage.
+
+Typically, you seek out the use of Presto when you experience an intensely slow
+query turnaround from your existing Hadoop, Spark, or Hive infrastructure. In
+fact, the genesis of Presto came about due to these slow Hive query conditions
+at Facebook back in 2012. So when you learn that Presto has a Hive connector,
+it can be rather confusing since you moved to Presto to circumvent the slowness
+of your current Hive cluster. Another common source of confusion is when you
+want to query your data from your S3 compatible object storage, such as AWS S3, 
+MinIO, and Google Cloud Storage. This too uses the Hive connector. If that 
+confuses you, don't worry, you are not alone. This blog aims to explain this
+commonly confusing nomenclature.
+
+<!--more-->
+
+When discussing the major components of the systems that emerged from big data
+architectures, you'll notice most have _runtime_, _storage_, and _metastore_
+components. While these components form the pillars of the architecture, an
+important component at the core of the Hive connector is the undocumented Hive
+_metastore specification_ describing how to map files to tables with columns
```

Comment:
I would call this something other than metastore, because that is the term used for their server.  Maybe "data organization specification"